resources/.eslintrc-react.js

```diff
@@ -65,16 +87,14 @@ module.exports = {
         "react/jsx-no-comment-textnodes": "error",                                    // Prevent comments from being inserted as text nodes
         "react/jsx-no-duplicate-props": "error",                                      // Prevent duplicate props in JSX
         "react/jsx-no-literals": "off",                                               // Prevent usage of unwrapped JSX strings
-        "react/jsx-no-target-blank": "off",                                           // Prevent usage of unsafe target='_blank'
+        "react/jsx-no-target-blank": "error",                                         // Prevent usage of unsafe target='_blank'
```

Comment:
what makes a target _blank unsafe?  I've forgotten.

Reply:
Recommendation is to make sure `rel="noopener noreferrer"` is added in conjunction with `target="blank"` links: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md because the opened page has full control over the `window` object of the page that linked it :)

Reply:
Excellent!